libsplinter/src/service/network/mod.rs

```diff
@@ -11,3 +11,579 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+//! The service network modules provides structs for managing the connections and communications
+//! with services processors over connections.
+
+mod error;
+
+use std::collections::{BTreeMap, HashMap};
+use std::sync::mpsc::{self, Receiver, Sender};
+use std::thread;
+
+use crate::network::connection_manager::{ConnectionManagerNotification, Connector};
+
+use self::error::ServiceConnectionAgentError;
+pub use self::error::ServiceConnectionError;
+
+/// Constructs new ServiceConnectionManager structs.
+///
+/// This builder is used to constructs a ServiceConnectionManager. At build time, this has
```

Comment:
used to constructs  -> used to construct

Reply:
Removed the sentence - it was redundant.